### DISCUSSION TITLE
Redirect management plugin access logs to console?

### DISCUSSION THREAD

**Discussion by rayate2410** [rabbitmq/rabbitmq-server#7759] [Questions]
As per https://www.rabbitmq.com/logging.html#logging-to-console, 

To activate console logging, use the following config snippet:
```log.console = true```

Above enables the rabbitmq logs to redirect to console.

I want the management API access logs to be redirected to console. 
As per https://www.rabbitmq.com/management.html#http-logging, we can redirect it to a file.
```management.http_log_dir = /path/to/folder```

But did not find option to redirect it to console.

**Comment by michaelklishin** [comment]
As far as I know the HTTP server we embed (Cowboy) only supports logging to a file.

**Comment by michaelklishin** [comment]
So it turns out, the request logging code is a part of RabbitMQ and could be updated to use stdout.

**Comment by michaelklishin** [comment]
Filed an issue https://github.com/rabbitmq/rabbitmq-server/issues/7764 but we won't get to it in the next month in all likelihood.

**Reply by rayate2410** [reply]
Thanks..
